[2025-04-08T22:31:39.323+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-08T22:31:40.482+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.create_raw_schema manual__2025-04-08T22:30:16.970994+00:00 [queued]>
[2025-04-08T22:31:40.705+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.create_raw_schema manual__2025-04-08T22:30:16.970994+00:00 [queued]>
[2025-04-08T22:31:40.706+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-08T22:31:41.176+0000] {taskinstance.py:2888} INFO - Executing <Task(SQLExecuteQueryOperator): create_raw_schema> on 2025-04-08 22:30:16.970994+00:00
[2025-04-08T22:31:41.447+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=216) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-08T22:31:41.336+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'create_raw_schema', 'manual__2025-04-08T22:30:16.970994+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/driven_data_pipeline.py', '--cfg-path', '/tmp/tmp_lm3abdb']
[2025-04-08T22:31:41.453+0000] {standard_task_runner.py:72} INFO - Started process 227 to run task
[2025-04-08T22:31:41.454+0000] {standard_task_runner.py:105} INFO - Job 10: Subtask create_raw_schema
[2025-04-08T22:31:41.787+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.create_raw_schema manual__2025-04-08T22:30:16.970994+00:00 [running]> on host 5a8aeda502d0
[2025-04-08T22:31:43.477+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='create_raw_schema' AIRFLOW_CTX_EXECUTION_DATE='2025-04-08T22:30:16.970994+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-08T22:30:16.970994+00:00'
[2025-04-08T22:31:43.545+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-08T22:31:44.373+0000] {sql.py:266} INFO - Executing: CREATE SCHEMA IF NOT EXISTS driven_raw;
[2025-04-08T22:31:45.197+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-04-08T22:31:47.524+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-04-08T22:31:47.631+0000] {sql.py:509} INFO - Running statement: CREATE SCHEMA IF NOT EXISTS driven_raw;, parameters: None
[2025-04-08T22:31:47.919+0000] {job.py:229} INFO - Heartbeat recovered after 10.81 seconds
[2025-04-08T22:31:48.665+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-08T22:31:48.679+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=create_raw_schema, run_id=manual__2025-04-08T22:30:16.970994+00:00, execution_date=20250408T223016, start_date=20250408T223140, end_date=20250408T223148
[2025-04-08T22:31:50.462+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-08T22:31:50.778+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-08T22:31:50.934+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
